app/views/users/show.html.erb

```diff
@@ -1 +1,15 @@
-<p>User profile</p>
\ No newline at end of file
+<section>
+<%= render 'user', user: @user %>
+  <div>
+    <h3>Bio</h3>
+    <p><%= @user.bio %></p>
+  <div>
+  <div>
+    <% @posts.each_with_index do |post, index| %>
+      <%= render partial: "posts/post" , locals: { post: post, index: index } %>
+    <% end %>
+  </div>
+   <button class="button">
+  See all posts
+  </button>
```

Comment:
- [ ] It will be good if this button will be clickable and show all posts. Kindly add it.

- [ ] I cant see all posts with writing this URL. Kindly fix it. [Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_all_posts.png)

![image](https://user-images.githubusercontent.com/38635158/174163242-9b018e61-fd0f-428e-80e0-94234c17f432.png)

Reply:
thanks for the review.i have made necessary changes